src/Form/EventFormType.php

```diff
@@ -10,55 +11,240 @@
+            ->add('organizerContact', TextType::class, ['label' => 'Контакты организаторов для обратной связи с Ролендарем'])
+            ->add(
+                'settlement',
+                EntityType::class,
+                [
+                    'class' => Settlement::class,
+                    'choice_label' => 'name',
+                    'label' => 'Тип поселения'
+                ]
+            )
+            ->add(
+                'weapons',
+                EntityType::class,
+                [
+                    'class' => Weapon::class,
+                    'choice_label' => 'name',
+                    'multiple' => true,
+                    'expanded' => true,
+                    'label' => 'Тип оружия',
+                    'required' => false
+                ]
+            )
+            ->add('priceMin', IntegerType::class,
+                [
+                    'label' => 'Размер денежного взноса от',
+                    'required' => false
+                ]
+            )
+            ->add('priceMax', IntegerType::class,
+                [
+                    'label' => 'Размер денежного взноса до',
+                    'required' => false
+                ]
+            )
+            ->add('contactSite', TextType::class,
+                [
+                    'label' => 'Сайт мероприятия',
+                    'required' => false
+                ]
+            )
+            ->add('contactFB', TextType::class,
+                [
+                    'label' => 'Группа в Facebook',
+                    'required' => false
+                ]
+            )
+            ->add('contactVK', TextType::class,
+                [
+                    'label' => 'Группа в Вконтакте',
+                    'required' => false
+                ]
+            )
+            ->add('contactTelegram', TextType::class,
+                [
+                    'label' => 'Телеграм',
+                    'required' => false
                 ]
             )
-            ->add('organizers', TextType::class)
-            ->add('organizerContact', TextType::class)
-            ->add('contactSite', TextType::class)
-            ->add('contactFB', TextType::class)
-            ->add('contactOther', TextType::class)
-            ->add('picture', FileType::class, array('label' => 'Add picture'))
-            ->add('save', SubmitType::class)
+            ->add('contactOther', TextType::class,
+                [
+                    'label' => 'Другие ссылки и контакты',
+                    'required' => false
+                ]
+            )
+            ->add('description', TextareaType::class, ['label' => 'Описание'])
+//            todo: actual demands for image attachment
+            ->add('picture', FileType::class,
+                [
+                    'label' => 'Загрузить картинку',
+                    'help' => 'Формат/размер/соотношение сторон',
```

Comment:
@daria-sieroshtan 
I think you want to replace this with actual thing

Reply:
In todo

Reply:
still in todo